pkgs/development/compilers/passerine/default.nix

```diff
@@ -2,16 +2,16 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "passerine";
-  version = "0.9.2";
+  version = "0.9.3";
 
   src = fetchFromGitHub {
     owner = "vrtbl";
     repo = "passerine";
-    rev = "dd8a6f5efc5dcb03d45b102f61cc8a50d46e8e98";
-    sha256 = "sha256-/QzqKLkxAVqvTY4Uft1qk7nJat6nozykB/4X1YGqu/I=";
```

Comment:
btw, what is this style of sha256? I never saw it written like this before

Reply:
It is a [Subresource Integrity (SRI)](https://www.w3.org/TR/SRI/) hash, where the hashing algo is part of the hash representation. The second part is the actual hash in base64.

SRI hashes are supported by many fetchers in the nix ecosystem now. But not all! `fetchpatch` is a notable exception at this point in time.